src/message/mimebody.rs

```diff
@@ -112,6 +112,20 @@ impl SinglePart {
         SinglePartBuilder::new()
     }
 
+    /// Directly create a `SinglePart` from an plain UTF-8 content
+    pub fn plain<T: IntoBody>(body: T) -> Self {
+        Self::builder()
+            .header(header::ContentType::TEXT_PLAIN)
+            .body(body)
+    }
+
+    /// Directly create a `SinglePart` from an UTF-8 HTML content
+    pub fn html<T: IntoBody>(body: T) -> Self {
```

Comment:
Not sure about which level to put these on, but I think it would be good to have shortcuts for these.

Reply:
Looks good